quantum/quantum.c

```diff
@@ -288,9 +288,11 @@ bool process_record_quantum(keyrecord_t *record) {
 
     if (record->event.pressed) {
         switch (keycode) {
+#ifndef NO_RESET
```

Comment:
XD002 has no physical reset button... 🤔

Reply:
Oops, pressed request changes by mistake. These are just comments.

Reply:
No reset button, just a bootloadhid style "hold this switch while plugging in". Main issue is even with these changes, if you enable rgb you only have about 70 bytes free. So RESET keycode when you only have 2 switches and zero layer capability... thats a luxury for those that chose a better mcu.